### PR TITLE
Add Security Audit Results to MINA FAQ 

### PR DESCRIPTION
**Pull request by ejMina226**
Now that MINA Protocol has successfully completed security audits, we would like to share the results with the community.
Preview here https://docs2-git-fork-ejmina226-patch-1-minadocs.vercel.app/about-mina/faq#what-security-audits-have-been-completed

### CODE REVIEW

File: docs/mina-faq.mdx [hunk 1]
```diff
@@ -303,3 +303,11 @@ this architecture emerged: https://www.youtube.com/watch?v=eWVGATxEB6M
 #### How do you control or limit the number of cores Snark Workers use?
 
 You can use the `-snark-worker-parallelism` flag when starting up `mina daemon`.
+
+## Security
+
```
Comment (barriebyron) [inline docs/mina-faq.mdx:308]:
```suggestion

#### What security audits have been completed?

You can review the results of the completed security audits here:

```

**Review by barriebyron — COMMENTED** [review]
@ejMina226 looks good with the suggested changes, find out about the Notion page sharing and we'll come back to that resource

**Review by nicc — APPROVED** [review]

File: docs/about-mina/faq.mdx [hunk 2]
```diff
@@ -34,3 +34,16 @@ The Mina node can run with an optional flag to make it an archive node. This mea
 Mina Protocol uses a custom proof system called Kimchi, which was developed by [O(1) Labs](https://o1labs.org/), and is the only blockchain offering infinite recursion.
 
 Check out the [Mina Book](https://o1-labs.github.io/proof-systems/plonk/overview.html) to learn more about Kimchi and the cryptography powering Mina Protocol.
+
+### What security audits have been completed?
+
+You can review the results of the completed security audits here:
+
+- On 2020-05-11 [NCC Group Security Audit Results of Mina Protocol](https://minaprotocol.com/blog/ncc-group-security-audit-results-of-coda-protocol) ([https://research.nccgroup.com/wp-content/uploads/2020/05/NCC_Group_O1Labs_O1LB001_Report_2020-05-11_v1.1.pdf](https://research.nccgroup.com/wp-content/uploads/2020/05/NCC_Group_O1Labs_O1LB001_Report_2020-05-11_v1.1.pdf))
```
Comment (barriebyron) [inline docs/about-mina/faq.mdx:42]:
```suggestion
- On 2020-05-11 [NCC Group Security Audit Results of Mina Protocol](https://research.nccgroup.com/wp-content/uploads/2020/05/NCC_Group_O1Labs_O1LB001_Report_2020-05-11_v1.1.pdf)
```

**Review by barriebyron — APPROVED** [review]
Good work @ejMina226 on your first docs PR!

**Pull request merged**